packages/preview-middleware-client/src/flp/bootstrap.ts

```diff
@@ -0,0 +1,34 @@
+import { getError } from '../cpe/error-utils';
```

Comment:
have to remove this again, it leads to "sap is not defined"

Reply:
next problem: the import from from 'sap/open/ux/preview/global' led to an export / ESM error at run-time. 
Replaced by a local definition.

Reply:
Both things solved, but now facing
![image](https://github.com/user-attachments/assets/b0f0d45e-98a7-4d3d-b665-53f878847793)

Reply:
done: had to move the call of initConnectors